composer.json

```diff
@@ -22,6 +22,7 @@
     "easyframework/collections": "dev-master"
   },
   "require-dev": {
+    "php-amqplib/php-amqplib": "2.6.*",
```

Comment:
When I'm using this package in `business` for example, `php-amqplib` won't be installed because it's in the dev requirements of `reagieren` and the suggestion will be useful for that.